fromconfig_mlflow/launcher.py

```diff
@@ -33,49 +31,55 @@ class MlFlowLauncher(fromconfig.launcher.Launcher):
       experiment
     - `artifact_location`: the location of the artifacts (config files)
 
-    Additionally, if you wish to call the `mlflow` launcher multiple
-    times during the launch (for example once before the parser, and
-    once after), you need to configure the different launches with the
-    special `launches` key (otherwise only the first launch will
-    actually log artifacts and parameters).
-
-    The `launches` key should be a list of dictionaries with the
-    following parameters
-
-    - `log_artifacts`: if `True` (default), will log the artifacts (the
-      config and command given to the launcher)
-    - `log_parameters`: if `True` (default) will log a flattened view of
-      the parameters
-    - `path_config`: if given, will write the config as an artifact with
-      that name (default is `config.json`)
-    - `path_command`: if given, will write the command as an artifact
-      with that name (default is `launch.txt`, using the `.txt`
-      extension because you can preview it on MlFlow).
-    - `include_keys`: if given, only log flattened parameters that have
-      one of these keys as substring. Also shorten the flattened
-      parameter to start at the first match. For example, if the config
-      is `{"foo": {"bar": 1}}` and `include_keys=("bar",)`, then the
-      logged parameter will be `"bar"`.
-    - `ignore_keys`: if given, parameters that have at least one of the
-      keys as substring will be ignored.
-    - `set_env_vars`: if given, mlflow environment variables are set, to
-      propagate the mlflow run id and tracking uri
-
-    Example
-    -------
-    >>> import fromconfig
-    >>> config = {
-    ...     "run": None,
-    ...     "launcher": {"log": ["logging", "mlflow"]},
-    ...     "mlflow": {"run_name": "test"}
-    ... }
-    >>> launcher = fromconfig.launcher.Launcher.fromconfig(config["launcher"])
-    >>> launcher(config, "run")
+    Attributes
+    ----------
+    log_artifacts : bool, optional
+        If True, save config and command as artifacts.
+    log_params : bool, optional
+        If True, log flattened config as parameters.
+    path_command : str, optional
+        Name for the command file
+    path_config : str, optional
+        Name for the config file.
+    set_env_vars : bool, optional
+        If True, set MlFlow environment variables.
+    set_run_id : bool, optional
+        If True, the run_id is overridden in the config.
+    ignore_keys : Iterable[str], optional
+        If given, don't log some parameters that have some substrings.
+    include_keys : Iterable[str], optional
+        If given, only log some parameters that have some substrings.
+        Also shorten the flattened parameter to start at the first
+        match. For example, if the config is `{"foo": {"bar": 1}}` and
+        `include_keys=("bar",)`, then the logged parameter will be
+        `"bar"`.
     """
 
+    def __init__(
```

Comment:
Ended up using the init, that's the most understandable way of parametrizing the launcher imo

Reply:
ok